File: Soumadip/Array/ArrayOperation.java
package Soumadip.Array;

import java.util.*;

public class ArrayOperation {
      // creating array
      public static void CreateArray(int k[]) {
            Scanner sc = new Scanner(System.in);
            for (int i = 0; i < k.length; i++) {
                  System.out.print("Enter number[" + i + "] : ");
                  k[i] = sc.nextInt();
            }
      }

      // display
      public static void display(int k[]) {
            System.out.print("\n\n[ ");
            for (int i = 0; i < k.length; i++) {
                  System.out.print(k[i] + "\t");

            }

            System.out.print("]\n\n");

      }

      // largest number
      public static void largest(int k[]) {
            int maxnum = Integer.MIN_VALUE;
            for (int i = 0; i < k.length; i++) {
                  if (maxnum < k[i]) {
                        maxnum = k[i];
                  }
            }
            System.out.println("\n\nThe largest number in the array is " + maxnum);
            System.out.println();
      }

      // smallest number
      public static void smallest(int k[]) {
            int minnum = Integer.MAX_VALUE;
            for (int i = 0; i < k.length; i++) {
                  if (minnum > k[i]) {
                        minnum = k[i];
                  }
            }
            System.out.println("\n\nThe smallest number in the array is " + minnum);
            System.out.println();
      }

      public static void main(String args[]) {
            int size;
            Scanner sc = new Scanner(System.in);

            System.out.println("Enter Element Size");
            size = sc.nextInt();

            int num[] = new int[size];
            CreateArray(num);
            display(num);
            largest(num);

            System.out.print("Do you want to see the smallest number? (1 = Yes): ");
            int choice;

            do {
                  choice = sc.nextInt();

                  if (choice != 1) {
                        System.out.print("Fuck i still waste your time choose to see smaleest number ");
                  }

            } while (choice != 1);

            smallest(num);

      }

}

File: Soumadip/Array/ArrayOperation.cs
using System;

namespace Soumadip.Array
{
    public class ArrayOperation
    {
        // creating array
        public static void CreateArray(int[] k)
        {
            for (int i = 0; i < k.Length; i++)
            {
                Console.Write("Enter number[" + i + "] : ");
                k[i] = Convert.ToInt32(Console.ReadLine());
            }
        }

        // display
        public static void display(int[] k)
        {
            Console.Write("\n\n[ ");
            for (int i = 0; i < k.Length; i++)
            {
                Console.Write(k[i] + "\t");

            }

            Console.Write("]\n\n");

        }

        // largest number
        public static void largest(int[] k)
        {
            int maxnum = int.MinValue;
            for (int i = 0; i < k.Length; i++)
            {
                if (maxnum < k[i])
                {
                    maxnum = k[i];
                }
            }
            Console.WriteLine("\n\nThe largest number in the array is " + maxnum);
            Console.WriteLine();
        }

        // smallest number
        public static void smallest(int[] k)
        {
            int minnum = int.MaxValue;
            for (int i = 0; i < k.Length; i++)
            {
                if (minnum > k[i])
                {
                    minnum = k[i];
                }
            }
            Console.WriteLine("\n\nThe smallest number in the array is " + minnum);
            Console.WriteLine();
        }

        public static void Main(string[] args)
        {
            int size;

            Console.WriteLine("Enter Element Size");
            size = Convert.ToInt32(Console.ReadLine());

            int[] num = new int[size];
            CreateArray(num);
            display(num);
            largest(num);

            Console.Write("Do you want to see the smallest number? (1 = Yes): ");
            int choice;

            do
            {
                choice = Convert.ToInt32(Console.ReadLine());

                if (choice != 1)
                {
                    Console.Write("Fuck i still waste your time choose to see smaleest number ");
                }

            } while (choice != 1);

            smallest(num);

        }

    }
}
